L1Trigger/DemonstratorTools/plugins/GTTFileWriter.cc

```diff
@@ -98,6 +104,10 @@ const std::map<size_t, l1t::demo::ChannelSpec> kChannelSpecs = {
     {48, {kTrackTMUX, 12, kGapLength}}, {49, {kTrackTMUX, 12, kGapLength}}, {50, {kTrackTMUX, 12, kGapLength}},
     {51, {kTrackTMUX, 12, kGapLength}}, {52, {kTrackTMUX, 12, kGapLength}}, {53, {kTrackTMUX, 12, kGapLength}}};
 
+const std::map<size_t, l1t::demo::ChannelSpec> kChannelSpecsOutputToCorrelator = {
+    /* channel index -> {link TMUX, TMUX index, inter-packet gap} */
+    {0, {kTrackTMUX, 0, kGapLength}}};
```

Comment:
As the TMUX period of the links of the correlator is 6, I think this line should instead be:
```
    {0, {6, 0, kGapLength}}};
```
In fact, we should probably add a corresponding static const above for this, e.g:
```
constexpr size_t kGTTBoardTMUX(6);
```
... and then update lines 128, 134 and 140 to re-use that number there (so that the code is a bit easier to understand - I now realise I should have put this constant in orginally). 

Do you agree, or have I misunderstood?

Reply:
Okay, I've done as you suggested. I made similar changes to the `GTTFileReader` (naming the constants). For both files I also moved the constants from the global scope into the class scope.